BaseTools/Edk2ToolsBuild.py

```diff
@@ -30,7 +31,7 @@ def ParseCommandLineOptions(self):
         # MU_CHANGE
         ParserObj.add_argument("-s", "--skip_path_env", dest="skip_env", default=False, action='store_true',
                                help="Skip the creation of the path_env descriptor file")
-        ParserObj.add_argument("-a", "--target_arch", dest="arch", default='IA32', choices=['IA32', 'ARM'],
+        ParserObj.add_argument("-a", "--target_arch", dest="arch", default=None, choices=[None, 'IA32', 'X64', 'ARM', 'AARCH64'],

```

Comment:
Should there still be a default, just for backward compatability?

Reply:
This is because the `None` was the default behavior for Linux build, where the make script will figure out the host system architecture and build target based on that. So `None` is essentially the backward compatibility option :). I will add some comments for this.